client/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import {
+  Form, Input, Button,
+} from 'antd';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import './style.css';
+import { setUser } from '../../redux/features/auth/user';
+
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const dispatch = useDispatch();
+  const { Item } = Form;
+  const { Password } = Input;
+  const changeEmail = (e) => {
+    e.preventDefault();
```

Comment:
Do you need to prevent the default here and when changing the password? you are not submitting the form here.